#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("max_depth_change_factor", double_t, 0, "max depth change factor", 0.02, 0.0, 1.0)
gen.add("normal_smoothing_size", double_t, 0, "normal smoothing size parameter", 10.0, 0.0, 100.0)
gen.add("estimation_method", int_t, 0, "estimation method", 0, 0, 2)
gen.add("depth_dependent_smoothing", bool_t, 0, "use depth dependent smoothing", False)
gen.add("border_policy_ignore", bool_t, 0, "ignore border policy", False)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "NormalEstimationIntegralImage"))
